commands/build.sh

```diff
@@ -76,6 +76,18 @@ if [[ "$(plugin_read_config NO_CACHE "false")" == "false" ]] ; then
     # The variable with this name will hold an array of group names:
     cache_image_name="$(service_name_cache_from_var "$service_name")"
 
+    # NOTE: customization to allow quick skipping for previously built images
```

Comment:
In case you weren't aware, there is also a diff here (both in this comment, and also in the code below), relative to `master` on our fork. Although again GitHub is not showing it (and is making it look like this code is all-new), you can see that a version of this code that already exists in `master` and the difference between that and this via: `git diff origin/master..reintegrate-changes commands/build.sh`.

I guess that I would tend to favor sticking with the code that we already have in `master` since we know that works? If you think that the code that you have here is better, though, then that's fine with me, too.

Regarding the comment, the version already in `master` is a bit more detailed / calls out a caveat regarding multi-image build steps:

> If we're using prior images and we find an exact match, terminate and skip all further processing. This won't work for a multi-image build step, only a single service can be built this way

Reply:
The thing we're losing on master is something I did to try to make our original PR mergeable upstream. It's accommodating a use case (building multiple images in one step) that we don't do. It's probably worth keeping around, I'll bring those changes back in.

Reply:
> The thing we're losing on master is something I did to try to make our original PR mergeable upstream. It's accommodating a use case (building multiple images in one step) that we don't do. It's probably worth keeping around, I'll bring those changes back in.

Ah, I see -- thank you for the info! 🙇‍♂️ 👍